libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ViewOptionsCustomization.kt

```diff
@@ -71,6 +73,16 @@ class ViewOptionsCustomization {
      */
     var infoPanelForcedState: Int? = null
 
+    /**
+     * Set to override [DistanceFormatterOptions]
+     */
+    var distanceFormatterOptions: DistanceFormatterOptions? = null
+
+    /**
+     * Set to override [DistanceFormatter]
+     */
+    var distanceFormatter: DistanceFormatter? = null
+
```

Comment:
Do we really need both? I believe having `distanceFormatterOptions` should be enough. 
With two options it is possible to specify contradicting values, and it is not clear how that would work.

Reply:
@Zayankovsky This is fixed.